pennylane/transforms/batch_partial.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Contains the batch dimension transform.
+"""
+import functools
+import inspect
+
+import pennylane as qml
+
+
+def _convert_to_args(func, args, kwargs):
+    """
+    Given a function, convert the positional and
+    keyword arguments to purely positional arguments.
+    """
+    sig = inspect.signature(func).parameters
+
+    new_args = []
+    for i, param in enumerate(sig):
+        if param in kwargs:
+            # first check if the name is provided in kwargs
+            new_args.append(kwargs[param])
+        elif i < len(sig):
```

Comment:
It's a typo. Actually, we don't even need that check since this function is only called within `batch_partial`, and we guarantee that it's called correctly.